Exporting these produces a broken config, since Macports moves these libs into a dedicated Python prefix.

--- ng/CMakeLists.txt	2023-12-23 05:42:55.000000000 +0800
+++ ng/CMakeLists.txt	2024-01-04 04:28:05.000000000 +0800
@@ -45,7 +45,7 @@
         set_target_properties( ngpy PROPERTIES OUTPUT_NAME "libngpy")
     endif()
     set_target_properties(ngpy PROPERTIES INSTALL_RPATH "${NG_RPATH_TOKEN}/../${NETGEN_PYTHON_RPATH}")
-    install(TARGETS ngpy DESTINATION ${NG_INSTALL_DIR_PYTHON}/${NG_INSTALL_SUFFIX} COMPONENT netgen EXPORT netgen-targets)
+    install(TARGETS ngpy DESTINATION ${NG_INSTALL_DIR_PYTHON}/${NG_INSTALL_SUFFIX} COMPONENT netgen)
 
     if(USE_GUI)
         add_library(ngguipy SHARED ngguipy.cpp)
@@ -57,7 +57,7 @@
             set_target_properties( ngguipy PROPERTIES OUTPUT_NAME "libngguipy")
         endif()
         set_target_properties(ngguipy PROPERTIES INSTALL_RPATH "${NG_RPATH_TOKEN}/../${NETGEN_PYTHON_RPATH}")
-        install(TARGETS ngguipy DESTINATION ${NG_INSTALL_DIR_PYTHON}/${NG_INSTALL_SUFFIX} COMPONENT netgen EXPORT netgen-targets)
+        install(TARGETS ngguipy DESTINATION ${NG_INSTALL_DIR_PYTHON}/${NG_INSTALL_SUFFIX} COMPONENT netgen)
     endif(USE_GUI)
 endif(USE_PYTHON)
 
